src/config/settings.py

```
@@ -2,7 +2,7 @@
 from pathlib import Path
 from typing import Any
 
-from pydantic_settings import BaseSettings
+from pydantic_settings import BaseSettings, SettingsConfigDict
```

The file imports SettingsConfigDict here. Important: the settings class below defines SECRET_KEY_ACCESS and SECRET_KEY_REFRESH defaults using os.urandom(32) (lines later in the file). os.urandom returns bytes, but your settings fields are annotated as str; this mismatch will break code that expects string keys (JWT libraries, env parsing, etc.). Change the defaults to produce strings, e.g.:

- os.urandom(32).hex()
- or base64.b64encode(os.urandom(32)).decode('ascii')
- or better: do not default to random values and require an env var in production, but use a deterministic test key in TestingSettings.

Please update the defaults at SECRET_KEY_ACCESS / SECRET_KEY_REFRESH accordingly to avoid bytes/str issues. See the SECRET_KEY lines in the same file for context (src/config/settings.py) .